lib/matplotlib/blocking_input.py

```diff
@@ -135,7 +135,7 @@ def mouse_event(self):
             self.mouse_event_pop(event)
         elif button == self.button_stop:
             self.mouse_event_stop(event)
-        else:
+        elif self.button_add:
```

Comment:
Shouldn't this be `elif button == self.button_add:`?

Or am I overlooking something and there is actually an asymmetry that there are other buttons that should be implicitly be handled as add event if `button_add` is defined?